cleancode/src/main/java/com/epam/cleancode/comments/MortgageInstallmentCalculator.java

```diff
@@ -2,38 +2,19 @@
 
 public class MortgageInstallmentCalculator {
 
-    /**
-     *
-     * @param p principal amount
-     * @param t term of mortgage in years
-     * @param r rate of interest
-     * @return monthly payment amount
-     */
-    public static double calculateMonthlyPayment(
-            int p, int t, double r) {
-
-        //cannot have negative loanAmount, term duration and rate of interest
-        if (p < 0 || t <= 0 || r < 0) {
+    public static double calculateMonthlyPayment(int principal, int mortgageTermInYears, double rateOfInterest) {
```

Comment:
Something wrong with this Pull Request. I shouldn't have changes in clean code module. Please recreate it and update link in issue